wcomponents-theme/src/main/js/wc/ajax/Trigger.js

```diff
@@ -191,7 +192,7 @@ function(tag, event, serialize, Widget, getAncestorOrSelf, ajax, formUpdateManag
 	 * @returns {Element} The element which is expected to fire the trigger.
 	 */
 	function getElement(trigger) {
-		var result, element;
+		let result, element;
 		if (trigger.id) {
 			if ((element = document.getElementById(trigger.id))) {
```

Comment:
I would have moved element here, still as a `let` and pulled it out of the if test + assignment, like this:

```javascript
let element = document.getElementById(trigger.id);
if (element) {
```

It's not really worth changing unless you feel like it, and you probably decided just to go minimal.

Reply:
I actually made that kind of change in one or two other places. I'll update it.

Reply:
I'm sure this is obvious but just to ensure I don't mislead you...

The test + assignment actually never made sense here (at the top level), but it does make some sense in the `else if` blocks below. 

I guess the obvious alternative would be to not do the whole "single return function" thing and have a bunch of returns in there instead of `else if` blocks.